main/src/ca/uwaterloo/flix/language/fmt/FormatType.scala

```diff
@@ -351,6 +352,9 @@ object FormatType {
 
       case SimpleType.Tuple(elms) =>
         elms.map(visit(_, Mode.Type)).mkString("(", ", ", ")")
+
+      case SimpleType.Error => "Error"
```

Comment:
Should this be something more distinctive?

Reply:
I think once `Type.Error` is actually used/shows up somewhere it would be sensible to do something better here.